blocks/editable/format-toolbar/index.js

```diff
@@ -42,55 +37,32 @@ const DEFAULT_CONTROLS = [ 'bold', 'italic', 'strikethrough', 'link' ];
 
 class FormatToolbar extends Component {
 	constructor( props ) {
-		super( ...arguments );
-		this.state = {
-			linkValue: props.formats.link ? props.formats.link.value : '',
-			isEditingLink: false,
-		};
+		super( props );
```

Comment:
I guess we should always prefer `super( ...arguments )` to avoid errors when we use the `context` feature?